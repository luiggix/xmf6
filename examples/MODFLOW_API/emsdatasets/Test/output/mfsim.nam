# File generated by Flopy version 3.9.1 on 02/10/2025 at 11:29:40.
BEGIN options
END options

BEGIN timing
  TDIS6  flow.tdis
END timing

BEGIN models
  gwf6  flow.nam  flow
END models

BEGIN exchanges
END exchanges

BEGIN solutiongroup  1
  ims6  flow.ims  flow
END solutiongroup  1

