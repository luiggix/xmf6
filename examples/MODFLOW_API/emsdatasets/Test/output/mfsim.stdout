                                   MODFLOW 6
                U.S. GEOLOGICAL SURVEY MODULAR HYDROLOGIC MODEL
                            VERSION 6.6.0 12/20/2024

   MODFLOW 6 compiled Dec 31 2024 17:12:23 with Intel(R) Fortran Intel(R) 64
   Compiler Classic for applications running on Intel(R) 64, Version 2021.7.0
                             Build 20220726_000000

This software has been approved for release by the U.S. Geological 
Survey (USGS). Although the software has been subjected to rigorous 
review, the USGS reserves the right to update the software as needed 
pursuant to further analysis and review. No warranty, expressed or 
implied, is made by the USGS or the U.S. Government as to the 
functionality of the software and related material nor shall the 
fact of release constitute any such warranty. Furthermore, the 
software is released on condition that neither the USGS nor the U.S. 
Government shall be held liable for any damages resulting from its 
authorized or unauthorized use. Also refer to the USGS Water 
Resources Software User Rights Notice for complete use, copyright, 
and distribution information.

 
 MODFLOW runs in SEQUENTIAL mode
 
 Run start date and time (yyyy/mm/dd hh:mm:ss): 2025/02/10 11:36:27
 
 Writing simulation list file: mfsim.lst
 Using Simulation name file: mfsim.nam
 
